schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals (
+ ID                              INT               PRIMARY KEY       NOT NULL,
+ NAME                       CHAR(50)     NOT NULL,
+ DATE_OF_BIRTH       DATE            NOT NULL,
+ ESCAPE_ATTEMPTS   INT              NOT NULL,
+ NEUTERED            VARCHAR(1)    NOT NULL,
```

Comment:
- [ ] As stated by the previous reviewer, kindly use `boolean` for NEUTERED column because this value will be true or false, so in this case boolean is a good option, it will be like this: `NEUTERED boolean,`